pkg/backend/display/progress.go

```diff
@@ -80,10 +81,16 @@ type DiagInfo struct {
 	// diagnostics for a resource.
 	//
 	// Diagnostic events are bucketed by their associated stream ID (with 0 being the default
-	// stream)
+	// stream).
 	StreamIDToDiagPayloads map[int32][]engine.DiagEventPayload
 }
 
+// PolicyInfo bundles policy information for a single resource.
+type PolicyInfo struct {
```

Comment:
do we need this wrapper?  can e not just store an array of PolicyPayloads per resource?

Reply:
I think struct makes sense in conjunction with some of the coming config changes.

Reply:
Ok.  Next question is: does the struct ever need to be represented as a pointer/nil?  that extra state of 'is this nil' versus 'is this empty' does not seem to buy us anything (that i can see).

Reply:
I chatted with Chris and I'm going to implement your first suggestion, thank you!